src/components/playback-bar/index.js

```diff
@@ -129,8 +130,40 @@ const PlaybackBar = ({ isLoading, onPreload, onSeek, onChange }) => {
     setSelectedIncrement(
       playbackIncrementForSelectedDuration(selectedTimeRange)
     );
+    setNerdletState((prevState) => ({
+      flow: {
+        ...prevState.flow,
+        pb_selectedTimeRange: selectedTimeRange,
```

Comment:
Let's not use cryptic variables in `nerdletState`. Use something like `playbackSelectedTimeRange` or if you want to keep it shorter, something like `playbackTime`. [Also stay away from nesting](https://github.com/newrelic/nr-labs-pathpoint/pull/498/files#r1970488864).